Included AllergyIntolerance, Patient, MedicationStatement and Device resources.

slvsridhar opened the pull request:
Updated slide bar to include AllergyIntolerance, Patient, MedicationStatement and Device resources.
Extension has been added for Patient and MedicationStatement resource.
Informational message added for device resource.
Incorporated the same changes which mentioned for Condition and DiagnosticReport resource.

daliboz commented:
Missing the individuals page/link is broken

daliboz commented on content/soarian/dstu2/medications.md:
```diff
@@ -0,0 +1,8 @@
+
+<%= render '/medications.*' %>
+* [MedicationStatement](../medications/medication-statement)
+* [Immunization](../medications/immunization)
```
Comment this out, since link will be broken

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-6453b2fe5bb289173ffc5f3650c243ce

daliboz commented on layouts/soarian_dstu2_sidebar.html:
```diff
@@ -20,7 +20,7 @@
         </h3>
         <ul class="js-guides">
-         <!--  <li><a href="/soarian/dstu2/general-clinical/allergy-intolerance/">AllergyIntolerance</a></li> -->
+         <li><a href="/soarian/dstu2/general-clinical/allergy-intolerance/">AllergyIntolerance</a></li> 
```
Need to uncomment the allergy in the general clinical page as well.

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-a5d2412080095fa0a003419d1ad4bf7f

daliboz commented on lib/resources/dstu2/soarian_allergy_intolerance.yaml:
```diff
@@ -0,0 +1,346 @@
+      note: RxNorm is supported for Allergens of medication category.
+    - display: SNOMED CT
+      system: http://snomed.info/
+      note: SNOMED CT is supported for negations.
```
Is it only for negations? Or is this also used for environmental/substance?

daliboz replied:
@slvsridhar @dougpratt ^^

slvsridhar replied:
Yes. It is only for negations and we clarified by adding text under the header of terminology bindings section

daliboz replied:
@slvsridhar What is used for non-med allergies like pollen?

slvsridhar replied:
For non-med allergies we are only sending the text.

daliboz commented on lib/resources/dstu2/soarian_allergy_intolerance.yaml:
```diff
@@ -0,0 +1,346 @@
+  note: resourceType must be AllergyIntolerance
+  url: http://hl7.org/fhir/DSTU2/json.html#resources
+
+- name: id
```
remove all non-binding related fields in this file. This service doesn't allow update/modify, so these aren't being used.

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-c2e4246bde1d063b55b859e76ed07b6e

daliboz commented on content/soarian/dstu2/general-clinical/allergy-intolerance.md:
```diff
@@ -0,0 +1,68 @@
+
+### Parameters
+     
+No AllergyIntolerance resource specific search parameters are supported. For an unambiguous patient-facing allergy list, the current allergy list is always returned; date date/range parameters are not used for this resource.
```
Remove this paragraph - this is already covered elsewhere.

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-814d617e2ca2dfd32c8fa2610c8aaaf1L34

daliboz commented on content/soarian/dstu2/general-clinical/allergy-intolerance.md:
```diff
@@ -0,0 +1,68 @@
+
+#### Request
+
+	GET https://fhir-myrecord.sandboxcerner.com/dstu2/123abc//AllergyIntolerance?patientId=52930295-B52D-4685-B922-DC5F181CF786
```
nit: remove the extra slash before AllergyIntolerance

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-814d617e2ca2dfd32c8fa2610c8aaaf1L49

daliboz commented on content/soarian/dstu2/general-clinical/allergy-intolerance.md:
```diff
@@ -0,0 +1,68 @@
+ 
+| Code | Message |
+| --- | --- |
+| PAPI001 | This is the most current, known allergy list and may not reflect the state of allergies known at a previous date.|
```
lowercase the "papi" portion of the code, it's lowercase in the actual error, and a code would typically be case-sensitive. Apply to both codes below.

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-814d617e2ca2dfd32c8fa2610c8aaaf1R62

daliboz commented on content/soarian/dstu2/devices/device.md:
```diff
@@ -0,0 +1,73 @@
+For the Device resource query, the API returns a list of charted procedures that include Unique Device Identifiers (UDIs) for the patient.  The list represents the most current procedure with UDI list that has been charted in Soarian.
+
+Note that traditional charting practices have procedures resolved and charted in coding/billing systems and not interfaced back to the EMR, therefore, the procedure record in Soarian may not be complete nor precise to a procedure list provided as part of a patient's billing record.  The list of procedures in completed or aborted state with UDIs will be as charted in Soarian but may not represent  ...
+Note that the data returned with the UDI may be incomplete or inconsistent because the FDA (Food and Drug Administration) requirement for UDI labelling, effective 2017, states that manufacturers are responsible for populating an FDA database (GUDID) with information related to their devices; this UDI labelling and GUDID population is a new process and inconsistencies and gaps may exist in the dat ...
```
Need an extra line break to break these paragraphs up.

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-05f4e9071af380fc012d35d174a66d4dR15

daliboz commented on content/soarian/dstu2/devices/device.md:
```diff
@@ -0,0 +1,73 @@
+Note that traditional charting practices have procedures resolved and charted in coding/billing systems and not interfaced back to the EMR, therefore, the procedure record in Soarian may not be complete nor precise to a procedure list provided as part of a patient's billing record.  The list of procedures in completed or aborted state with UDIs will be as charted in Soarian but may not represent  ...
+Note that the data returned with the UDI may be incomplete or inconsistent because the FDA (Food and Drug Administration) requirement for UDI labelling, effective 2017, states that manufacturers are responsible for populating an FDA database (GUDID) with information related to their devices; this UDI labelling and GUDID population is a new process and inconsistencies and gaps may exist in the dat ...
+The FHIR definition of device is broader than the ONC definition of implantable device.  The API Device response is per the ONC definition of those devices that are implanted and have a UDI.  They are charted per encounter, per procedure where a patient may have
+1 or more visits where each visit may have
```
Make this bulleted, add line break so that it formats correctly (it's all one big paragraph when rendered right now)

MaxPhilips replied:
Gave it a try as this:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-05f4e9071af380fc012d35d174a66d4dR18

Thoughts @daliboz ?

daliboz replied:
looks good but will need @slvsridhar and @dougpratt to validate.

slvsridhar replied:
Thanks @MaxPhilips .

Now it looks good.

daliboz commented on content/soarian/dstu2/devices/device.md:
```diff
@@ -0,0 +1,73 @@
+0 or more procedures charted in completed or aborted state where each procedure may have
+0 or more devices with UDIs charted
+
+Ambiguity might exist for intended contexts of date representing a comprehensive state / version of UDIs on a particular date or alternatively UDIs implanted / explanted / serviced on a particular date.  Therefore, to avoid ambiguity and per ONC direction, all UDIs charted in Soarian will be returned for any query and date/date range parameters will not be supported.  
```
I don't think this paragraph is needed, there is not date parameter documented below, so it's obvious that it's not supported.

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-05f4e9071af380fc012d35d174a66d4dL21

daliboz commented on lib/resources/dstu2/soarian_device.yaml:
```diff
@@ -0,0 +1,32 @@
+      note: Currently will return SNO code from the below system.
+      terminology:
+      - display: Identifier Type Codes
+        system: http://hl7.org/fhir/identifier-type
```
This should link to the DSTU2 set.

daliboz replied:
Is identifier type actually returned? I don't see it in the example below.

daliboz replied:
Still needs done @slvsridhar

daliboz replied:
@slvsridhar ?

slvsridhar replied:
No, In the current implementation it was not returning the identifier type. JIRA has been raised to fix this in a code and we have updated the documentation with appropriate identifier type.

slvsridhar replied:
Linked to DSTU2 set

daliboz commented on content/soarian/dstu2/devices/device.md:
```diff
@@ -0,0 +1,73 @@
+
+### Parameters
+  
+No Device resource specific search parameters are supported. To avoid ambiguity of date context and per ONC guidance, the current UDI list is always returned; date date/range parameters are not used for this resource.  
```
remove this paragraph.

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-05f4e9071af380fc012d35d174a66d4dL40

daliboz commented on content/soarian/dstu2/devices/device.md:
```diff
@@ -0,0 +1,73 @@
+
+#### Request
+	
+	GET https://fhir-myrecord.sandboxcerner.com/dstu2/123abc//Device?patientId=7568F2F3-FC76-4185-A540-1DB56331A387
```
remove extra slash

daliboz commented on lib/resources/dstu2/soarian_device.yaml:
```diff
@@ -0,0 +1,32 @@
+  description: The specific kind of device.
+  binding:
+    description: The type of device.
+    note: Currently not bound to any terminology and only 'Device.type.text' will be returned as of now.
```
Based on the example, this does not seem accurate.

daliboz replied:
@slvsridhar ?

slvsridhar replied:
Yes it is not accurate. JIRA has been raised to fix this in a code and we have updated the documentation with proper terminology bindings

daliboz commented on content/soarian/dstu2/devices/device.md:
```diff
@@ -0,0 +1,73 @@
+
+| Code | Message |
+| --- | --- |
+| PAPI014 | No implantable medical devices are associated with this query. |
```
lowercase the "papi" portion to match what is actually returned by the API

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-05f4e9071af380fc012d35d174a66d4dR70

daliboz commented on content/soarian/dstu2/medications/medication-statement.md:
```diff
@@ -0,0 +1,83 @@
+
+  Status Extension:
+
+The URL for this status extension is defined as `Additional status`
```
This doesn't match the id defined below, and it should. It also cannot have spaces: http://hl7.org/fhir/DSTU2/datatypes.html#id

Also, usually the URL is fully qualified. EG:
http://cerner.hs.fhir.com/StructureDefinition/additional-status

We should fix this in the implementation, and update the doc here.

daliboz replied:
@slvsridhar @dougpratt ^^

slvsridhar replied:
JIRA has been raised to fix this in the implementation. we have updated the documentation with the mentioned fully qualified URL

daliboz commented on content/soarian/dstu2/medications/medication-statement.md:
```diff
@@ -0,0 +1,83 @@
+
+ID                                | Value\[x] Type      | Description
+----------------------------------|---------------------|----------------------------------------------------------------------------------
+`MedicationStatement/Staus`          | [`string`]       | This element describes the additional status information of the MedicationStatement. Possible values are 'New', 'Continued' and 'Modified' |
```
Use the tick marks to make these possible values look like codes. Also, make sure the case is accurate (from the example, it looks like these should all be lowercase).

Is there a better description of this? What is the "additional status" describing? The status of reconciliation?

daliboz replied:
@slvsridhar

slvsridhar replied:
Used tick marks to make possible values to look like codes and changed to lowercase to match with the implementation.  
Modified the description about the additional status. 

Added additional statuses as an extension to properly portray the status of the medications on the DMR as the existing statuses do not reflect the correct status of the medication.

daliboz commented on content/soarian/dstu2/medications/medication-statement.md:
```diff
@@ -0,0 +1,83 @@
+
+ Example:
+ 
+<%= json(:SOARIAN_MEDICATIONSTATEMENT_STATUS_EXTENSION) %>
```
This is a less complex extension - don't need the example unless you really want to include it :)

daliboz replied:
@slvsridhar

slvsridhar replied:
Removed this example :-)

daliboz commented on content/soarian/dstu2/medications/medication-statement.md:
```diff
@@ -0,0 +1,83 @@
+
+### Parameters
+
+No MedicationStatement resource specific search parameters are supported. To ensure an unambiguous patient-facing active medication list, the current list is always returned; date date/range parameters are not used for this resource.  
```
Remove this - it's covered already elsewhere.

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-c4abc9b95b9475be23080ce2d68eb598L48

daliboz commented on content/soarian/dstu2/medications/medication-statement.md:
```diff
@@ -0,0 +1,83 @@
+
+#### Request
+
+	GET https://fhir-myrecord.sandboxcerner.com/dstu2/123abc//MedicationStatement?patientId=393EC770-CFC9-4F5A-8B3A-784F2508F245
```
remove extra slash

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-c4abc9b95b9475be23080ce2d68eb598R61

daliboz commented on content/soarian/dstu2/medications/medication-statement.md:
```diff
@@ -0,0 +1,83 @@
+---
+title: MedicationStatement | Soarian DSTU 2 API
```
May want to add a comment about usage of contained. Also, likely need to change the binding table since it doesn't look like medicationCodeableConcept can ever be returned and make this a contained binding? 
http://fhir.cerner.com/millennium/dstu2/medications/medication-statement/#contained-medication-bindings

daliboz replied:
@slvsridhar

slvsridhar replied:
Modified binding table to have only contained binding and added comment about contained bindings in the implementation notes section

daliboz commented on content/soarian/dstu2/medications/medication-statement.md:
```diff
@@ -0,0 +1,83 @@
+
+| Code | Message |
+| --- | --- |
+| PAPI023 | No relevant clinical data exists.|
```
lowercase "papi" portion

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-c4abc9b95b9475be23080ce2d68eb598R76

daliboz commented on lib/resources/dstu2/soarian_patient.yaml:
```diff
@@ -0,0 +1,32 @@
+      description: The type used to determine which identifier to use for a specific purpose.
+      terminology:
+      - display: Identifier Type Codes
+        system: http://hl7.org/fhir/identifier-type
```
both systems should probably link to DSTU2 specific URLs

daliboz replied:
@slvsridhar

slvsridhar replied:
Updated the link to DSTU2

daliboz commented on lib/resources/dstu2/soarian_patient.yaml:
```diff
@@ -0,0 +1,32 @@
+        system: http://hl7.org/fhir/v2/0203
+
+- name: contact
+  required: 'No'
```
remove contact, no binding provided

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-4ae0de069b68e676185064ebfba25932L24

daliboz commented on content/soarian/dstu2/individuals/patient.md:
```diff
@@ -0,0 +1,101 @@
+
+  Race Extension:
+
+ The URL for this custom extension is defined as  `http://hl7.org/fhir/StructureDefinition/us-core-race`
```
This isn't a custom extension. Just list it under Extensions. EG: http://fhir.cerner.com/millennium/dstu2/individuals/patient/#extensions

Rename section to "Extensions"

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-fa2d6f77a687ec62885939cc35682185R28

daliboz commented on content/soarian/dstu2/individuals/patient.md:
```diff
@@ -0,0 +1,101 @@
+
+### Parameters
+
+No Patient resource specific search parameters are supported. To ensure an unambiguous patient-facing response, the most current information is always returned; date date/range parameters are not used for this resource.
```
Remove this paragraph.

Also, I'm confused - it states no params are supported, but the example below shows a patientId param? If you can query by patientId, include that parameter in the table.

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-fa2d6f77a687ec62885939cc35682185L56

@dougpratt @slvsridhar I added a parameter table with patientId.  Let me know if this is incorrect

slvsridhar replied:
Thanks @MaxPhilips  
Parameter table looks perfect.
Patient resource uses patientId parameter.

daliboz commented on content/soarian/dstu2/individuals/patient.md:
```diff
@@ -0,0 +1,101 @@
+
+#### Request
+
+	GET https://fhir-myrecord.sandboxcerner.com/dstu2/123abc//Patient?patientId=393EC770-CFC9-4F5A-8B3A-784F2508F245
```
remove extra slash.

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-fa2d6f77a687ec62885939cc35682185R60

daliboz commented on content/soarian/dstu2/individuals/patient.md:
```diff
@@ -0,0 +1,101 @@
+
+The API returns the most current information about the patient including:
+
+| Name | Description | 
```
Remove the comment above, and this table. Make it a bulleted list in the overview. EG: http://fhir.cerner.com/millennium/dstu2/individuals/patient/#overview

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-fa2d6f77a687ec62885939cc35682185R14

daliboz replied:
@MaxPhilips No need to move these down to the lower table -they're never reused. They also need a target like in the Patient example so they open in a new window.

daliboz replied:
@MaxPhilips or @slvsridhar ^^

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/5cdc093dec456c176564dc4fd012d8e22dc67929

I left the links moved down below because it looks nicer and is more extensible

I did not set the Extensions link to open in a new tab because it just jumps to a heading on this page

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/449410f9eeed051cdd6dc21c1eac42b8a3b2e954

Changed reference links to inline links

Added new tab functionality to the US Core Race and Ethnicity links in the Extensions section

daliboz commented on content/soarian/dstu2/individuals/patient.md:
```diff
@@ -0,0 +1,101 @@
+
+One or more patient races and ethnicities will be returned as extensions, see extension section.
+
+* Note that the value for gender in the patient resource may or may not represent the value of sex explicitly recorded at birth.  ONC has clarified that birth sex is the CCDS (common clinical data set) value.  This information, if charted and configured is returned as part of the Observation resource.
```
Move this to an implementation note. You can reference the note along with the gender when the list is moved to the overview.

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-fa2d6f77a687ec62885939cc35682185R41

@dougpratt @slvsridhar Adding a reference to the updated implementation note from the overview list is an outstanding change

slvsridhar replied:
We have provided reference to the implementation notes.

daliboz commented on content/soarian/dstu2/individuals/patient.md:
```diff
@@ -0,0 +1,101 @@
+
+| Code | Message |
+| --- | --- |
+| PAPI003 | This is the most current, known Patient data and may not reflect the state of information known at a previous date.|
```
lowercase "papi" portion

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-fa2d6f77a687ec62885939cc35682185R75

daliboz commented on lib/resources/dstu2/soarian_medication_statement.yaml:
```diff
@@ -0,0 +1,202 @@
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
```
remove all fields that don't have associated bindings. Likely, this file gets removed completely and a contained medication yaml is created instead, with just the medication.code field bound.

daliboz replied:
@slvsridhar

slvsridhar replied:
Removed this yaml file and added contained medication yaml file.

daliboz commented on lib/resources/example_json/soarian_examples_ethnicity_status_extension.rb:
```diff
@@ -0,0 +1,21 @@
+module Cerner
```
Will be removed.

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-e51d1805f77407cf1744b7cb44de5693

daliboz commented on lib/resources/example_json/soarian_examples_medicationstatement_status.rb:
```diff
@@ -0,0 +1,17 @@
+module Cerner
```
Can be removed.

daliboz replied:
@slvsridhar

slvsridhar replied:
removed this example

daliboz commented on lib/resources/example_json/soarian_examples_race_extension.rb:
```diff
@@ -0,0 +1,24 @@
+module Cerner
```
will be removed.

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-09c3053a40b6a75922aa1ab5a50dbb04

daliboz commented on content/soarian/dstu2/individuals/patient.md:
```diff
@@ -0,0 +1,101 @@
+ The URL for this custom extension is defined as  `http://hl7.org/fhir/StructureDefinition/us-core-ethnicity`
+
+ID                                | Value\[x] Type      | Description
+----------------------------------|---------------------|----------------------------------------------------------------------------------
```
Same as above - this isn't custom, you just need to link to the main definition from HL7.

MaxPhilips replied:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-fa2d6f77a687ec62885939cc35682185R35

daliboz commented:
Done with review, pending changes and clarifications requested by comments.

Will still need 1 more reviewer.

CC @slvsridhar @dougpratt

MaxPhilips commented on content/soarian/dstu2/general-clinical/allergy-intolerance.md:
```diff
@@ -0,0 +1,68 @@
+
+For the AllergyIntolerance resource query, the API returns the current state of the patient's allergy list as charted in Soarian.   The allergy list represents a clinical assessment of a patient's allergy or intolerance when exposed to a specific substance or class of substance.  
+
+Soarian's list supports medication, food, environmental, and other allergen categories.  Allergy and Intolerance are not differentiated.  Charted assertions of no allergy is different than an empty list; the response will include any charted assertions such as NKA (no known allergies), NKDA (no known drug allergy), NKFA (no known food allergy), No Latex Allergy, and/or No IV contrast Allergy.  A  ...
```
"A consolidated list of allergens, reactions, and severities that providers have charted in the Soarian Allergy List."
Seems like an incomplete sentence

MaxPhilips replied:
Took a guess with:
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-814d617e2ca2dfd32c8fa2610c8aaaf1R14

@dougpratt @slvsridhar Can you double check my change?

slvsridhar replied:
Thanks @MaxPhilips . Sentence looks good.

MaxPhilips commented on content/soarian/dstu2/individuals/patient.md:
```diff
@@ -0,0 +1,101 @@
+---
```
Need to add an individuals.md landing page

MaxPhilips commented:
Added an individuals page
https://github.com/cerner/fhir.cerner.com/pull/133/commits/2d18ffc5b1ddbc31cb81aee13465180197b12519#diff-5b159d9b5d9286ad0d149656a419bfcb

MaxPhilips commented:
Pushed a commit addressing most of the comments so far.  I replied to all comments that I addressed - anything not replied to is outstanding.  Additionally, a few comments that I've replied to I'm looking for confirmation that the change is acceptable.

These comments are outstanding but now apply to changed lines, so they'll be hidden by default:
https://github.com/cerner/fhir.cerner.com/pull/133#discussion_r114140480
https://github.com/cerner/fhir.cerner.com/pull/133#discussion_r114140605

daliboz commented:
I tried to call out places we were still waiting on changes or input

slvsridhar commented:
Thanks @MaxPhilips for addressing the comments. 
We have addressed the outstanding comments and pushed a commit.

@daliboz Please review and share your thoughts.

daliboz commented on content/soarian/dstu2/medications/medication-statement.md:
```diff
@@ -17,32 +17,32 @@ To avoid patient confusion about the definitive active medication list, no in-pr
  ID                         | Value\[x] Type | Description
 ----------------------------|----------------|---------------------------------------------------------------------------------------------------------------------------------------------
-`MedicationStatement/Staus` | [`string`]     | This element describes the additional status information of the MedicationStatement. Possible values are 'New', 'Continued' and 'Modified'.
+`MedicationStatement/Staus` | [`string`]     | This element describes the additional status information of the MedicationStatement. Additional statuses has beed added to properly portray the status of the medications on the DMR as the existing statuses do not reflect the correct status of the medication.Possible values are `new`, `continued` and `modified`.
```
what's the DMR? Discharge medication reconciliation list?
Maybe (the first sentence isn't adding anything):

This element describes the status of the medications on the last discharge medication reconciliation list, rather than the order status. Possible values are `new`, `continued` and `modified`.

daliboz replied:
Update the id to additional-status and update the example response to match the URL documented.

slvsridhar replied:
DMR means Discharge medication reconciliation.

We have added clarification that additional status pertains to medication whose primary status is active.

slvsridhar replied:
We have updated the id to additional status and updated the example resource to match

daliboz commented on content/soarian/dstu2/medications/medication-statement.md:
```diff
@@ -17,32 +17,32 @@ To avoid patient confusion about the definitive active medication list, no in-pr
 
 * The API returns qualifying results dated within the implicit or explicitly defined date query parameter.  For response time, response may be limited to 1000 records.  For any non-null response, the API will provide the informational message including the record dates included in the response.  This informational message may be augmented with further detail if noted below.    
 * The Narrative includes additional information pertaining to the MedicationStatement, and should be shown to the user to ensure completeness of clinical content and context.
+* [MedicationStatement.medication] may be a reference to a [contained] Medication.  Medications in the system always exist within the context of a MedicationStatement and cannot be be referenced independently.
```
may be or are always?

slvsridhar replied:
We have updated from may be to 'is always'

daliboz commented on content/soarian/dstu2/general-clinical/allergy-intolerance.md:
```diff
@@ -15,6 +15,8 @@ Soarian's list supports medication, food, environmental, and other allergen cate
 
 ## Terminology Bindings
 
+Medication allergens are coded in RxNorm. Negations are indicated by SNOMED codes. All other allergens (e.g., environmental substances) are returned as text only.
```
Don't need this. The terminology table already had this. My confusion is what are non-med allergens coded as?

slvsridhar replied:
for non-med allergies we are returning only text.

daliboz commented on content/soarian/dstu2/individuals/patient.md:
```diff
@@ -38,10 +38,10 @@ The following extensions can also be returned:
 	GET /Patient?:parameters
 	
-_Implementation Notes_
+### Implementation Notes
```
This should be surrounded by underscores, not a heading. For the link above, just link to the section Implementation notes is immediately under.

daliboz replied:
fixed

slvsridhar replied:
Thanks for fixing it.

daliboz commented on content/soarian/dstu2/individuals/patient.md:
```diff
@@ -78,3 +78,4 @@ These additional informational messages may be returned within the [bundle exten
 
 [bundle extension]: ../../#bundle-message-extension
 [`reference`]: http://hl7.org/fhir/dstu2/search.html#reference
+[Implementation notes]: ../patient/#implementation-notes
```
remove this. Just link in line.

daliboz replied:
fixed

daliboz commented on content/soarian/dstu2/individuals/patient.md:
```diff
@@ -38,10 +38,10 @@ The following extensions can also be returned:
 
 * The API returns the most current patient list.
-* The value for gender in the patient resource may or may not represent the value of sex explicitly recorded at birth.  ONC has clarified that birth sex is the CCDS (common clinical data set) value.  This information, if charted and configured is returned as part of the Observation resource.
+* This is the Administrative Gender.  Birth Sex,  if charted and configured, is returned as part of the Observation resource.   
```
Should be "Returns the administrative gender only. Birth Sex, if charted and configured, is returned from the Observation resource.

daliboz replied:
fixed

daliboz commented:
I rebased the branch, brought the terminology formats up to date, and addressed the comments above, except for outstanding questions around the non-med allergies and I'm not sure what system is used for the Device type.

dougpratt commented:
@daliboz, our API returns coded values only for medication allergens; non-medication allergens are text only.  Is this contrary to certification requirements?  If so we'll have to move quickly; if not, it will be considered for a future enhancement.

daliboz commented:
@dougpratt  No, not required by cert. I added implementation notes and a note under the terminology table to clarify. Once we confirm all the other terminologies are correct, I'll push that commit for review.

daliboz commented:
This has been rebased/squashed and the last unreviewed commits are separated. With changes, +1 from me.

slvsridhar commented:
Thank you Jenni. We have reviewed the latest changes and it looks good.

MaxPhilips commented:
+1
